src/content/fr/2019/media.md

```diff
@@ -0,0 +1,416 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=167590779"
+  )
+}}
+
+### Images adaptatives
+
+Un autre axe d’amélioration des performances des pages consiste à utiliser des images adaptatives. Cette technique vise à réduire le nombre d’octets d’image en diminuant les pixels supplémentaires qui ne sont pas affichés à l’écran en raison du rétrécissement de l’image. Au début de ce chapitre, vous avez vu que la page web médiane sur ordinateur utilisait un MP d’espace réservé pour les images mais transférait 2,1&nbsp;MP de volume de pixels réel. Comme il s’agissait d’un test DPR 1x, 1,1&nbsp;MP de pixels ont été transférés sur le réseau, mais n’ont pas été affichés. Pour réduire cette surcharge, nous pouvons utiliser l’une des deux (éventuellement trois) techniques suivantes:
+
+* **Le balisage HTML** - utiliser une combinaison d’éléments `<picture>` et `<source>` avec les attributs `srcset` et `sizes` permet au navigateur de choisir la meilleure image à partir des dimensions du <span lang="en">viewport</span> et la densité de l’affichage. 
+* **Indices Client** - permet de déléguer le choix des images éventuellement redimensionnées à la négociation de contenu HTTP.
+* **BONUS**: des bibliothèques JavaScript pour retarder le chargement des images jusqu’à ce que le JavaScript puisse s’exécuter, inspecter le DOM du navigateur et injecter l’image correcte en fonction du conteneur.
+
+### Utilisation du balisage HTML
+
+La méthode la plus courante pour implémenter des images adaptatives est de construire une liste d’images alternatives en utilisant soit `<img srcset>` soit `<source srcset>`. Si le `srcset` est basé sur le DPR, le navigateur peut sélectionner la bonne image dans la liste sans information supplémentaire. Cependant, la plupart des implémentations utilisent également `<img sizes>` pour aider à indiquer au navigateur comment effectuer le calcul de mise en page nécessaire pour sélectionner l’image correcte dans le `srcset` basé sur les dimensions des pixels.
+
+{{ figure_markup(
+  image="fig14_html_usage_of_responsive_images.png",
+  caption="Pourcentage de pages utilisant des images adaptatives avec HTML.",
+  description="Un diagramme à barres montre que 18&nbsp;% des images utilisent `sizes`, 21&nbsp;% `srcset` et 2&nbsp;% `images`.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=582530039&format=interactive"
+  )
+}}
+
+L’utilisation nettement plus faible de `<picture>` n’est pas surprenante dans la mesure où cet élément est le plus souvent utilisé pour des gabarits adaptatifs avancés en termes de [direction artistique](https://developer.mozilla.org/fr/docs/Apprendre/HTML/Comment/Ajouter_des_images_adaptatives_%C3%A0_une_page_web#D%C3%A9cision_de_nature_artistique).
+
+### Utilisation de sizes
```

Comment:
sizes/`sizes`

Reply:
Same as lang, it totally breaks the anchors: `/fr/2019/media#utilisation-de-laquonbspsizesnbspraquo`